types/usb/index.d.ts

```diff
@@ -306,23 +310,23 @@ export class Interface {
    *
    * The device must be open to use this method.
    */
-  detachKernelDriver(): number;
+  detachKernelDriver(): void;
```

Comment:
Does this no longer return a number?

Reply:
actually it never returned a number, AFAIK [[relevant blame line 1]](https://github.com/tessel/node-usb/blame/5c0d7ff6cd147e5beb53236d7d47da67ff017773/src/device.cc#L302) [[relevant blame line 2]](https://github.com/tessel/node-usb/blame/a69f1e9cb83d11d88cb009013b7346f8c3eac3a4/usb.js#L326)

Reply:
ack